src/webapp/default_config.py

```diff
@@ -18,6 +18,8 @@
 CONTACT_DATA_BRANCH = "master"
 CONTACT_CACHE_LIFETIME = 60 * 2
 
+CILOGON_LDAP_PASS = None
```

Comment:
```suggestion
CILOGON_LDAP_PASSFILE = None
```
That's what you use in models.py.

Reply:
Good catch.